Include/cpython/code.h

```diff
@@ -90,6 +91,7 @@ struct PyCodeObject {
     int co_nplaincellvars;      /* number of non-arg cell variables */
     int co_ncellvars;           /* total number of cell variables */
     int co_nfreevars;           /* number of free variables */
+    int _co_obj_cache_len;      /* number of entries in _co_obj_cache */
```

Comment:
This is only ever used for an `assert`, right? It feels wasteful to include it here just for that purpose.

Reply:
Yes.
Generally, when allocating a variable sized array you need the length somewhere.
But it seems that in this case, we don't.